bin/mc-send-to-console

```diff
@@ -17,7 +17,7 @@ if [ ! -p "${CONSOLE_IN_NAMED_PIPE}" ]; then
   exit 1
 fi
 
-if [ "$(id -u)" = 0 ]; then
+if [ "$(id -u)" != 0 ]; then
   if [[ $(getDistro) == alpine ]]; then
     exec su-exec minecraft bash -c "echo '$*' > '${CONSOLE_IN_NAMED_PIPE:-/tmp/minecraft-console-in}'"
```

Comment:
...but you can't sudo to an arbitrary user from a non-root user

Reply:
oh you're right, i need thinkabout non-root user running.